src/rnp/fficli.cpp

```diff
@@ -1547,10 +1547,15 @@ cli_cfg_set_keystore_info(rnp_cfg_t *cfg)
     return true;
 }
 
-static ssize_t
-stdin_reader(void *app_ctx, void *buf, size_t len)
+static bool
+stdin_reader(void *app_ctx, void *buf, size_t len, size_t *readres)
 {
-    return read(STDIN_FILENO, buf, len);
+    int64_t res = read(STDIN_FILENO, buf, len);
```

Comment:
Same here, `ssize_t`.

Reply:
@dewyatt Ok. I'm not finished with this PR yet (changing interface to bool/size_t*), will let you know once done. Probably will make some force-pushes before that.